src/cmd/raw.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/tkhq/go-sdk/pkg/api/client/private_keys"
+	"github.com/tkhq/go-sdk/pkg/api/models"
+)
+
+var (
+	rawSignPayloadEncoding string
+	rawSignHashFunction    string
+	rawSignPayload         string
+)
+
+func init() {
+	rawCmd.Flags().StringVar(&signingKeyID, "signing-key", "", "name or ID of the signing key")
+
+	rawSignCmd.Flags().StringVar(&rawSignPayloadEncoding, "payload-encoding",
+		string(models.V1PayloadEncodingPAYLOADENCODINGTEXTUTF8), "encoding of payload")
+	rawSignCmd.Flags().StringVar(&rawSignHashFunction, "hash-function",
+		string(models.V1HashFunctionHASHFUNCTIONSHA256), "hash function")
+	rawSignCmd.Flags().StringVar(&rawSignPayload, "payload",
+		"", "payload to be signed")
+
+	rawCmd.AddCommand(rawSignCmd)
+
+	rootCmd.AddCommand(rawCmd)
+}
+
+var rawCmd = &cobra.Command{
+	Use:   "raw interacts with unstructured requests to the Turnkey API",
```

Comment:
I'm not sure "unstructured" is right. These requests are still highly structured, but this is "raw" because we're only providing pure curve signing, and the API requires users to know what they're doing with encoding / hash functions.

Reply:
Changed "unstructured" to "low-level".